test/cluster-isolation/mzcompose.py

```diff
@@ -226,7 +250,12 @@ def run_test(c: Composition, disruption: Disruption, id: int) -> None:
 
             validate(c)
 
-        cleanup_list = ["materialized", "testdrive", *[n.name for n in nodes]]
+        cleanup_list = [
+            "materialized",
+            "testdrive",
+            "postgres",
+            *[n.name for n in nodes],
+        ]
         c.kill(*cleanup_list)
         c.rm(*cleanup_list, destroy_volumes=True)
 
```

Comment:
As mentioned on Slack, you should be able to back out all the changes to the mzcompose files now!